<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="shubham-dhiman" failures="1" tests="2" name="Default test" time="27.824" errors="0" timestamp="2023-11-23T17:40:37 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.468"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.658">
    <failure type="java.lang.AssertionError" message="Forcefully failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Forcefully failed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at pageObjects.LoginPage.failedscenrios(LoginPage.java:74)
at stepDefinations.LoginPageStepDef.verify_failure(LoginPageStepDef.java:35)
at âœ½.verify failure(file:///C:/Users/shubham.dhiman/eclipse-workspace/JenkinsGitHubDemo/src/test/java/Features/1.Login.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
